regolith/builders/coabuilder.py

```diff
@@ -0,0 +1,173 @@
+"""Builder for Resumes."""
+
+import datetime as dt
+import os
+import sys
+import openpyxl
+
+from regolith.builders.basebuilder import BuilderBase
+from regolith.dates import month_to_int
+from regolith.sorters import doc_date_key, ene_date_key, position_key
+from regolith.tools import all_docs_from_collection, filter_publications, \
+    month_and_year, fuzzy_retrieval, is_since
+from copy import copy
+from dateutil.relativedelta import relativedelta
+
+
+
+def mdy_date(month, day, year, **kwargs):
+    if isinstance(month, str):
+        month = month_to_int(month)
+    return dt.date(year, month, day)
+
+
+def mdy(month, day, year, **kwargs):
+    return "{}/{}/{}".format(
+        str(month_to_int(month)).zfill(2), str(day).zfill(2), str(year)[-2:]
+    )
+
+
+class RecentCollaboratorsBuilder(BuilderBase):
+    """Build recent collaborators from database entries"""
+
+    btype = "recent-collabs"
+    needed_dbs = ['citations', 'people', 'contacts', 'institutions']
+
+    def __init__(self, rc):
+        super().__init__(rc)
+        self.template = os.path.join(
+            os.path.dirname(os.path.dirname(__file__)), "templates", "coa_template.xlsx"
+        )
+        self.cmds = ["excel"]
+
+    def construct_global_ctx(self):
+        super().construct_global_ctx()
+        gtx = self.gtx
+        rc = self.rc
+
+        gtx["people"] = sorted(
+            all_docs_from_collection(rc.client, "people"),
+            key=position_key,
+            reverse=True,
+        )
+        gtx["contacts"] = sorted(
+            all_docs_from_collection(rc.client, "contacts"),
+            key=position_key,
+            reverse=True,
+        )
+        gtx["institutions"] = all_docs_from_collection(rc.client,
+                                                       "institutions")
+        gtx["citations"] = all_docs_from_collection(rc.client, "citations")
+        gtx["all_docs_from_collection"] = all_docs_from_collection
+
+    def excel(self):
+        rc = self.rc
+        gtx = self.gtx
+        since_date = dt.date.today() - relativedelta(months=48)
```

Comment:
let's remove the hard-coded 48

Reply:
Are we going to declare a global variable at the top like NMONTHS = 48 and then pass that into the excel function?